usaspending_api/api_contracts/contracts/v2/agency/toptier_code/obligations_by_type.md

```diff
@@ -0,0 +1,58 @@
+FORMAT: 1A
+HOST: https://api.usaspending.gov
+
+# Federal Obligations [/api/v2/agency/{toptier_code}/obligations_by_type/{?fiscal_year}]
+
+This endpoint is used to power USAspending.gov's agency profile pages.
+
+## GET
+
+This endpoint returns a breakdown of obligations by award type (contracts, IDVs, grants, loans, direct payments, other) within a fiscal year.
```

Comment:
AFAIK "Award Category" is mostly a derived value in USAspending from the Award Type Code values associated with each award.

"Award Category" may be more accurate and less confusing than "Award Type" in descriptions. I realize `award_type` is established in other endpoints so this is just a conversation

Reply:
Good point, we also use "award type" for A, B, C, etc. and "award category" for contracts, grants, etc. on the frontend